### PR TITLE
feat: add tag search

### PR DESCRIPTION
**Pull request by azayz**

### CODE REVIEW

File: deployment/bff/app/v1/routers/txt2img.py [hunk 1]
```diff
@@ -55,12 +55,14 @@ def search(data: NowTextSearchRequestModel):
     """
     Retrieve matching images for a given text as query.
     """
-    query_doc = process_query(text=data.text, uri=data.uri)
+    query_doc, filter_query = process_query(
+        blob=data.image, uri=data.uri, conditions=data.conditions
```
Comment (joschkabraun) [inline deployment/bff/app/v1/routers/txt2img.py:59]: use `text` instead of `blob`

File: deployment/bff/app/v1/routers/text2text.py [hunk 2]
```diff
@@ -49,12 +49,14 @@ def search(data: NowTextSearchRequestModel):
     """
     Retrieve matching text for a given text as query.
     """
-    query_doc = process_query(text=data.text, uri=data.uri)
+    query_doc, filter_query = process_query(
+        blob=data.image, uri=data.uri, conditions=data.conditions
```
Comment (joschkabraun) [inline deployment/bff/app/v1/routers/text2text.py:53]: use `text` instead of `blob`

**Review by joschkabraun — APPROVED** [review]
See minor typos and bump docker tag. Otherwise LGTM!

**Review by makram93 — APPROVED** [review]
LGTM!

**Pull request merged**